core/commands.go

```diff
@@ -94,6 +94,16 @@ var (
 		Arity:    2,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	jsontypeCmdMeta = DiceCmdMeta{
+		Name: "JSON.TYPE",
+		Info: `JSON.TYPE key [path]
+		Returns string reply for each path, specified as the value's type.
+		Returns RespNIL If the key doesn't exist.
+		Error reply: If the number of arguments is incorrect.`,
+		Eval:     evalJSONTYPE,
+		Arity:    -1,
+		KeySpecs: KeySpecs{BeginIndex: 1},
+	}
```

Comment:
Arity will be -2, right?

Also please make add this new object to the bottom of the list (we run an automated script to generate docs and it requires new objects to be added to the bottom of the list)

Reply:
yes. I misunderstand the arity is args number here. I will fix it immediately

Reply:
done